src/Actions/actions.js

```diff
@@ -65,6 +65,23 @@ function getPosts(posts) {
     }
 }
 
+function updateClickedImage(currentClickedImage){
+    return {
+        type: "UPDATE_CURRENT_CLICKED_IMAGE",
+        data: currentClickedImage
+    }
+}
+
+function handleClickedImage(currentClickedImage, images){
```

Comment:
it's nicer to use named params, `({currentClickedImage, images})`.

Also, these names don't really tell me much, could be more descriptive